public-ui/src/app/views/ehr-view/ehr-view.component.ts

```diff
@@ -38,6 +38,7 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   private searchRequest: SearchConceptsRequest;
   private subscriptions: ISubscription[] = [];
   private initSearchSubscription: ISubscription = null;
+  showMoreSynonyms = {};
```

Comment:
nit: would add a comment here describing what this property does

Reply:
Done thanks.